docs/tutorials/ruby-driver-transactions.txt

```diff
@@ -0,0 +1,73 @@
+============
+Transactions
+============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+Version 4.0 of the MongoDB server introduces
+`multi-document transactions <https://docs.mongodb.com/manual/core/transactions/>`_.
+(Updates to multiple fields within a single document are atomic in all
+versions of MongoDB.) Ruby driver version 2.6.0 adds support for transactions.
+
+Using Transactions
+------------------
+
+In order to start a transaction, the application must have a :ref:`session <sessions>`.
+
+A transaction can be started by calling the ``start_transaction`` method on a session:
+
+.. code-block:: ruby
+
+  session = client.start_session
+  session.start_transaction
+
+It is also possible to specify read concern, write concern and read preference
+when starting a transaction:
+
+.. code-block:: ruby
+
+  session = client.start_session
+  session.start_transaction(
+    read_concern: {level: :majority},
+    write_concern: {w: 3},
+    read: {mode: :primary})
+
+A transaction must be committed or aborted. The corresponding methods
```

Comment:
> A transaction must be committed or aborted

This isn't technically true; when a session ends (either because the user explicitly ends it or the session is garbage collected), any transaction in progress will be implicitly aborted. I think it's better to phrase this as "may be committed or aborted" and then to move the sentence below about implicit aborts here.